src/app/course/event/course-events-snippet/course-events-snippet.component.ts

```diff
@@ -125,6 +126,10 @@ export class CourseEventsSnippetComponent implements OnInit {
      * @param courseId - The course you are importing the event into.
      */
     importCourseEvent(event: CourseEvent, courseId: number): void {
+        const holdFeatured: boolean = event.featured
+        if(holdFeatured){
+            this.clearFeatured(event.id)
+        }
```

Comment:
This should be done in the back-end. It's actually easy to do. When importing an event you can just set the featured to be `None`.